### PR TITLE
More render tests

### PR DESCRIPTION
**Pull request by henrikrudstrom**
depends on #84
adds tests for anchoring, 
AnchorsFill.qml fails, probably because it references a sibling.

### CODE REVIEW

**Comment by ChALkeR** [comment]
Not sure why the existing tests png images were changed. Is that required. They were smaller (optimized), btw. Could you remove those changes, please?

The actual changes look good, but I would prefer to keep the opacity for now until we decide on that one.

Would it be fine to you if I would rebase this, remove changes present in #84 (those could be discussed separately) and add opacity for now?

File: tests/QMLEngine/scope.js [hunk 1]
```diff
@@ -19,4 +19,10 @@ describe('QMLEngine.scope', function() {
     expect(parentItem.sum).toBe(6600);
     div.remove();
   });
+
+  it('can reference sibling items by id', function() {
+    var qml = loader('Parent', this.div);
```
Comment (ChALkeR) [inline tests/QMLEngine/scope.js]: Does that file even exist?

Comment (henrikrudstrom) [inline tests/QMLEngine/scope.js]: which file?

Comment (ChALkeR) [inline tests/QMLEngine/scope.js]: SiblingParent.qml

Comment (ChALkeR) [inline tests/QMLEngine/scope.js]: ScopeParent.qml

Comment (henrikrudstrom) [inline tests/QMLEngine/scope.js]: should be loader(´Sibling´... good your eyes are working at least

**Comment by henrikrudstrom** [comment]
strange, PropertyGrid test (the one with transparent reference image) still passes even when the screenshot code adds white background...

**Comment by henrikrudstrom** [comment]
cleaned up and rebased, @ChALkeR, @pavelvasev, @Plaristote LGTY?

**Comment by ChALkeR** [comment]
Rebased against master, optimized images, minor style fixes.
@henrikrudstrom Is everything ok?

**Comment by ChALkeR** [comment]
Ok, LGTM. Landing.

**Pull request merged**